mobile/android/android-debug-bridge.ts

```diff
@@ -0,0 +1,54 @@
+///<reference path="../../../.d.ts"/>
+"use strict";
+
+import util = require("util");
+
+export class AndroidDebugBridge implements Mobile.IAndroidDebugBridge {
+	constructor(private identifier: string,
+		private $childProcess: IChildProcess,
+		private $errors: IErrors,	
+		private $logger: ILogger,	
+		private $staticConfig: Config.IStaticConfig) { }
+	
+
+	public executeCommand(...args: string[]): IFuture<any> {
+		return (() => {
+			let command = this.composeCommand(args).wait();
+			this.$childProcess.exec(command).wait();
```

Comment:
return type is `IFuture<any>`, but you are not returning anything, maybe missing return before this childProcess exec or incorrect return type